Lab_17_RaceCars/src/GasCar.ts

```diff
@@ -0,0 +1,23 @@
+import Racer from "./Racer";
+
+export default class GasCar implements Racer {
+    team: string 
+    speed: number = 0
+    fuel: number 
```

Comment:
When a class implements an interface, you don't have to explicitly annotate types inside of the class. That's part of the convenience of interfaces. This applies to `SolarCar` as well.
```suggestion
export default class GasCar implements Racer {
```

Reply:
It's not a bad thing to annotate a class which implements an interface. There's just a tradeoff to consider: more convenient and concise or more explicit?